index.html

```diff
@@ -1,19 +1,127 @@
 <!DOCTYPE html>
 <html>
-
 <head>
 	<title>PizzaParty</title>
 	<link href="css/style.css" rel="stylesheet">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```

Comment:
these two lines need to switch per https://getbootstrap.com/docs/5.0/getting-started/introduction/#css